src/components/Footer.js

```diff
@@ -24,17 +36,42 @@ export function Versions({ loading, error, bblfshdVersion, webClientVersion }) {
     return <span>{error}</span>;
   }
 
+  const languageVersions = Object.keys(languages).reduce((acc, k) => {
+    const { version, name } = languages[k];
+
+    if (version === undefined) {
```

Comment:
When this happens?

Reply:
When the `(auto)` language is found. I changed this to make it more clear in 8d31e67.